watch/app/src/main/java/com/google/samples/quickstart/engagesdksamples/watch/data/model/TestData.kt

```diff
@@ -26,13 +27,15 @@ class TestData() {
           id = "$i",
           movieName = "Title $i",
           landscapePoster = R.drawable.red,
+          platformType = PlatformType.TYPE_UNSPECIFIED,
```

Comment:
Let's use something other than `UNSPECIFIED` like `ANDROID_TV`

Reply:
Changed to TYPE_ANDROID_TV